contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -71,78 +90,156 @@ parameters:
       - input: dstIP
         output: dstLocation
         type: add_location
-    type: network
-- extract:
+- name: extract_conntrack
+  extract:
+    type: conntrack
+    conntrack:
+      keyDefinition:
+        fieldGroups:
+        - name: src
+          fields:
+          - srcIP
+          - srcPort
+        - name: dst
+          fields:
+          - dstIP
+          - dstPort
+        - name: protocol
+          fields:
+          - proto
+        hash:
+          fieldGroupRefs:
+          - protocol
+          fieldGroupARef: src
+          fieldGroupBRef: dst
+      outputRecordTypes:
+      - newConnection
+      - flowLog
+      - endConnection
+      outputFields:
+      - name: bytes_total
+        operation: sum
+        input: bytes
+      - name: bytes
+        operation: sum
+        splitAB: true
+      - name: packets_total
+        operation: sum
+        input: packets
+      - name: packets
+        operation: sum
+        splitAB: true
+      - name: numFlowLogs
+        operation: count
+      - name: timeStart
+        operation: min
+        input: timeReceived
+      - name: timeEnd
+        operation: max
+        input: timeReceived
+      endConnectionTimeout: 30s
+- name: extract_aggregate
+  extract:
+    type: aggregates
     aggregates:
     - name: bandwidth_network_service
       by:
       - service
+      - _RecordType
       operation: sum
```

Comment:
Should we use this syntax ( _ prefix) also for other variables defined internally, such as `aggregate` and `by` in the prom stage?

Reply:
We can open that for a discussion. I'm not against. @eranra @jotak WDYT?

Reply:
+1 but this can be done outside of this PR